api/v1alpha1/local_object_reference_types.go

```diff
@@ -0,0 +1,110 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// ServicesDefaultLocalObjectReference identifies an API object within a
+// known namespace that defaults resource to services if unspecified.
```

Comment:
You may also want to mention that it's also defaulting group here and for the ConfigMapRef.

Reply:
ack